drivers/ieee802154/ieee802154_dw1000.c

```diff
@@ -84,6 +84,9 @@ struct dwt_phy_config {
 	uint8_t tx_shr_code;	/* TX SHR preamble code */
 	uint32_t tx_shr_nsync;	/* PLEN index, e.g. DWT_PLEN_64 */
 
+	bool phr_mode_ext;	/* Extended PHR mode: long frame 1024 Byte */
```

Comment:
I agree this in terms of complying with the 15.4 standard. However, I would argue that the device has the capability of transmitting long frame. Also, there is a setting `CONFIG_IEEE802154_RAW_MODE` for the user to choose not totally comply with 15.4 and directly process the data by the user. Having the choice doesn't mean the driver will not comply with 15.4.

Reply:
This device has capability of so much, but it does not make sense to add something just because it is possible. I do not know why CONFIG_IEEE802154_RAW_MODE is relevant here or not comply to anything.

Reply:
In my previous project, we have used long frame and smart TX power.  Anyway, I can remove these two extensions to comply with IEEE802.15.4 standard. Thanks for the comments. 

How about the runtime configuration API for changing the channel configuration? Any comments?